sdk/src/main/java/io/radar/sdk/RadarFraudManager.kt

```diff
@@ -0,0 +1,51 @@
+package io.radar.sdk
+
+import android.content.Context
+import android.util.Log
+import com.google.android.play.core.integrity.IntegrityManagerFactory
+import com.google.android.play.core.integrity.IntegrityTokenRequest
+
+internal class RadarFraudManager(
+    private val context: Context,
+    private val verificationSettings: RadarVerificationSettings?,
+    private val logger: RadarLogger,
+) {
+
+    fun getIntegrityToken(block: (integrityToken: String?) -> Unit) {
```

Comment:
wondering if we want to add the status as part of this function signature:

```
block: (status, integrityToken) -> Unit
```

we can crib off something like: https://github.com/1nikolas/play-integrity-checker-app/blob/main/app/src/main/java/gr/nikolasspyr/integritycheck/MainActivity.java#L223-L278

that would inform what to do for the TODO below.

we can check that status returned later in `track`